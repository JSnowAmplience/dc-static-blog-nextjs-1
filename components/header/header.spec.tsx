/* eslint-env jest */
import renderer from 'react-test-renderer';
import Header from './header';

describe('Header', () => {
  test('renders blog header', async () => {
    const props = {
      title: 'Blog Title'
    };
    const component = await renderer.create(<Header {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
