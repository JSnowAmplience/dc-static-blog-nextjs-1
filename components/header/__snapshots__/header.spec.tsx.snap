// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders blog header 1`] = `
<header
  className="jsx-1124965052"
>
  <section
    className="jsx-1124965052"
  >
    <img
      alt="Amplience logo"
      src="/static/images/logo-380w.png"
      srcSet="/static/images/logo-190w.png 1x, /static/images/logo-380w.png 2x"
    />
    <h1
      className="jsx-1124965052"
    >
      Blog Title
    </h1>
  </section>
</header>
`;
